src/praisonai-agents/praisonaiagents/memory/memory.py

```
@@ -635,36 +635,49 @@ def store_short_term(
-                    "content": text,
-                    "metadata": metadata,
-                    "created_at": datetime.utcnow(),
-                    "memory_type": "short_term"
-                }
-                self.mongo_short_term.insert_one(doc)
-                logger.info(f"Successfully stored in MongoDB short-term memory with ID: {ident}")
+                result_id = self.memory_adapter.store_short_term(text, metadata=metadata)
+                logger.info(f"Successfully stored via memory adapter with ID: {result_id}")
+                adapter_success = True
+                ident = result_id  # Use adapter-provided ID
             except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)